README.md

```diff
@@ -18,99 +18,112 @@ On an on-going basis there will be:
 - Capistrano redeploys to the demo server
 - and swaps of the production and demo servers.
 
-For more details, see `scripts/deploy.sh`: Ideally, this will run as-is,
-but you could also read it as instructions, and do each step manually.
+For more details, see `scripts/deploy.sh` or review documentation below.
 
-## Ingest PBCore
+## Deploy, Server Swap and Ingest Requirements
+In order to deploy code to the website, swap servers from demo to live and/or ingest PBCore xml you'll need two additional repositories.
 
-The authoritative metadata records for the site come from a FileMaker database which exports a zip
-of PBCore xml files, with one description document per file. No further cleaning of the PBCore
-is done as part of ingest, in contrast to the AAPB.
+[openvault3_deploy](https://github.com/WGBH/openvault3_deploy)
 
-For development the fixtures may be enough:
+[aws-wrapper](https://github.com/WGBH/aws-wrapper)
 
-```
-$ ruby scripts/ingest.rb --stdout-log --same-mount --dirs spec/fixtures/pbcore
-```
+Make sure you first check out these two repositories and pull the latest code.
+
+For WGBH Open Vault server resources such as ssh keys, urls, AWS site names, and AWS Zone IDs, please see [Server Resources](https://wiki.wgbh.org/display/MLA/Server+Resources) documentation on the internal wiki.
 
-The ingester can read the zip export directly: Unzipping is unnecessary.
+You should also have Amazon Web Services command line interface installed and configured.
+[See the WGBH Stock Sales readme](https://github.com/WGBH/stock-sales-2#uploading-to-amazon-s3) for AWS setup instructions.
 
+
+If you have all the required applications and dependencies, a good first test would be to see if you can obtain the ip addresses for the current live and demo Open Vault servers.
+
+Open your Terminal application.
 ```
-$ ruby scripts/ingest.rb --stdout-log --same-mount --files ~/Downloads/export.zip
+$ cd /aws-wrapper
+$ ruby scripts/ssh_opt.rb
```

Comment:
- You almost certainly won't have /aws-wrapper at root.
- You're imagining that all the necessary params have been put in the defaults.yml?